cloudformation/application.yml

```diff
@@ -0,0 +1,405 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: PostgREST Starter Kit Application Service
+Parameters:
+  ListenerHostNamePattern:
+    Type: String
+    Description: "Listen for requests on the load balancer for this domain. The  'loadbalancer.yml' stack must have already been created to use this stack template."
+    Default: "*.mydomain.com"
+  HasHttpsListener:
+    Type: String
+    Default: "No"
+    AllowedValues: [ "Yes", "No" ]
+    Description: "Select this if a Certificate ARN was provided when creating The 'loadbalancer.yml' stack."
+  ClusterName:
+    Type: String
+    Description: "Name of the ECS cluster created with the wizard"
+  ECRRegion:
+    Type: String
+    Description: "Load Docker images from this AWS ECR region. If omitted, the current region is used."
+  SandboxPostgresImage:
+    Type: String
+    Default: "sandbox-pg:latest"
+    Description: "This image exists in your account's ECR repository."
+  OpenRestyImage:
+    Type: String
+    Default: "openresty:latest"
+    Description: "This image exists in your account's ECR repository."
+  PostgrestImage:
+    Type: String
+    Default: "subzerocloud/postgrest"
+    Description: "This image is not prepended with the ECR host because it is a standard PostgREST image."
+  DesiredCount:
+    Type: Number
+    Default: "1"
+    Description: "Attempt to start this many application instances"
+  SandboxPostgresSuperUser:
+    Type: String
+    Default: "admin"
+  SandboxPostgresSuperUserPassword:
+    Type: String
+    Default: "adminpass"
+  DbHost:
+    Type: String
+  DbPort:
+    Type: String
+    Default: "5432"
+  DbName:
+    Type: String
+    Default: "app"
+  DbSchema:
+    Type: String
+    Default: "api"
+  DbUser:
+    Type: String
+    Default: "authenticator"
+  DbPassword:
+    Type: String
+    Default: "authenticatorpass"
+  DbAnonRole:
+    Type: String
+    Default: "anonymous"
+  DbPool:
+    Type: String
+    Default: "10"
+  MaxRows:
+    Type: String
+    Description: "MAX_ROWS env var"
```

Comment:
should this have a default value "" so that you don't need to specify it at setup?